<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(17, 122, 101);</string>
  </property>
  <widget class="QWidget" name="centralwidget_p">
   <widget class="QStackedWidget" name="stackedWidget_p">
    <property name="geometry">
     <rect>
      <x>-150</x>
      <y>100</y>
      <width>1061</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="produit">
     <widget class="QLabel" name="label_2_p">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 bold italic 25pt &quot;Calibri&quot;;
background-color: rgb(17, 122, 101);</string>
      </property>
      <property name="text">
       <string>Gestion Du Stock</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_p">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>341</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3_p">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Libelle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nbre_Exemplaires</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id_Fournisseur</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Id_p">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Nom_p">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prix_p">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbre_E_p">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>320</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Idf_p">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>380</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_Libelle_p">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(39, 39, 39);
</string>
       </property>
       <item>
        <property name="text">
         <string>Alimentaires</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entretients</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_17_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: 75 bold italic 14pt &quot;Calibri&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Produit:</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_p">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>691</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_Ajouter_p">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifier_p">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimer_p">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_10_p">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>530</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Recherche:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="croissant_p">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>440</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>croissant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9_p">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>450</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>trier par:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Tri_p">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>450</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nombre d'exemplaires</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prix </string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="decroissant_p">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>470</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>decroissant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4_p">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>530</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6_p">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>560</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt; precedent</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="statistique_p">
     <widget class="QLabel" name="label_11_p">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>page stats</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView_p">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="alerte_p">
     <widget class="QLabel" name="label_12_p">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Alertes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13_p">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier seuil de l'alerte</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_p">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="alerter_p">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>alerter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mailing">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mailing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Objet:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="a_mail">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="objet_mail">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="message_mail">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="send_p">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Quitter_p">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Stat_p">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistiques </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_alertes_p">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alertes </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_map_p">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Map des rayons </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mail_p">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mailing </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_p">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 bold italic 11pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Voir aussi:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
